module/applications/global/prompt-factory.mjs

```diff
@@ -433,7 +479,20 @@ class ItemPromptFactory extends PromptFactory {
   }
 
   async _talentCategoryPrompt() {
-    const buttons = Object.entries( ED4E.talentCategory ).map(
+
+    const versatilityId = game.settings.get( "ed4e", "edidVersatility" );
+    const actor = this.document.actor;
+    let category = ED4E.talentCategory;
+    if ( actor.items.filter( item => item.system.edid === versatilityId ).length === 0 ) {
+      if ( typeof category === "object" && category !== null ) {
```

Comment:
Why is all that in here? Are you trying to remove versatility from the categories? What for?

Reply:
hmm, okay. I think the entire `if ( !versatility) {...}` block can be reduced to something like this:
```javascript
const versatilityItem = this.document.actor.getSingleItemByEdid( ... );
const { versatility, ...categoriesWithoutVersatility } = ED4E.talentCategory;

const buttons = Object.entries( versatilityItem ? ED4E.talentCategory : categoriesWithoutVersatility  ).map(...)
```

Reply:
I don't get that

Reply:
ok